aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -133,11 +133,11 @@ fft_norm_mode norm_from_string(c10::optional<c10::string_view> norm, bool forwar
 
 // Fixes the shape of x such that x.size(dims[i]) == sizes[i],
 // either by zero-padding, or by slicing x starting from 0.
-Tensor resize_fft_input(Tensor x, IntArrayRef dims, IntArrayRef sizes) {
+Tensor resize_fft_input(Tensor x, IntArrayRef dims, SymIntArrayRef sizes) {
```

Comment:
IIRC sizes are symbolic here because they are compared to x_sizes later, I'll double-check.